include/fg/CPUCopy.hpp

```diff
@@ -0,0 +1,43 @@
+/*******************************************************
```

Comment:
ok, i am fine with moving them out. Just saying that 'include' is also forge header location so i don't understand your point.